components/layout_2020/table/layout.rs

```diff
@@ -667,12 +688,12 @@ impl<'a> TableLayout<'a> {
         // > * If the table-root has 'width: auto', the used width is the greater of min(GRIDMAX,
         // >   the table’s containing block width), the used min-width of the table.
         let used_width_of_table = match content_box_size.inline {
```

Comment:
I think the comment above is inaccurate now? Let's just remove the part about what to do if the table has a computed inline size and write something like:

```rust
// This diverges from the specification a little:
// If table-root has a computed value for inline size: Use the max of the minimum inline content size
// of the grid items and the inline size specified on the table style. The inline size specified on table
// style is already expressed via the content block size for children.
```

Reply:
But there is always a computed value for inline size...

I have refactored this function and tweaked the comments, please check.